src/features/Auth/model/actions/authThunk.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  createUserWithEmailAndPassword,
+  onAuthStateChanged,
+  signInWithEmailAndPassword,
+  signOut,
+} from "@firebase/auth";
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+import { auth } from "../../../../firebase";
+import { authActions } from "../slices/authSlice";
+
+interface Data {
```

Comment:
```suggestion
interface Credentials {
```
как вариант. Больше смысла.

Reply:
Окей